src/plugins/event_annotation/public/event_annotation_service/service.tsx

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import { EventAnnotationServiceType } from './types';
+import { defaultAnnotationColor } from '..';
```

Comment:
It seems to me that it is not very correct to do such an import, there is clearly a cross reference here.  

With tree shaking, in such cases a duplicate is created and this leads in some cases to errors